### PR TITLE
Qa

### LINKED ISSUE #38: [FR] Export HTML code to external files and add #NOQA for those
**Is your feature request related to a problem? Please describe.**
Currently we get complains by flake that our html code exceeds line length.  However, there exist no better way of dealing with this as it is still readable but due to the structure of html naturally gets quite lengthy.

**Describe the solution you'd like**
Move the layout code to an separate python file and import it in the corresponding page. Then ignore those external files in the flake config, so that we get proper QA for the "functional" code and skip QA for the "structural" one.

**Describe alternatives you've considered**
Open to other suggestions..

**Additional context**
N/A

### LINKED ISSUE #41: [FR] Add QA
chore.

### PR DESCRIPTION
**Pull request by stroblme**
This PR resolves #41 and resolves #38

**Depends on #44**

### CODE REVIEW

**Comment by stroblme** [comment]
Just thought it would have been smarter to first address #38 , because flake now complains a lot.. well know what to do now..

**Comment by stroblme** [comment]
checks failing because of #cirKITers/qml-essentials/pull/62 . Would suggest to merge as licensecheck issues are being resolved when upgrading qml essentials

**Review by majafranz — CHANGES_REQUESTED** [review]
Apart from the missing elements in the side bar, it looks good. Thanks for the great work!

File: app/app.py [hunk 1]
```diff
@@ -1,151 +1,19 @@
...
 import sys
 
 import logging
 
+
 app = dash.Dash(
     external_stylesheets=[dbc.themes.BOOTSTRAP, dbc.icons.FONT_AWESOME], use_pages=True
 )
+
+from layouts.app_page_layout import sidebar, content
```
[the remark this answers is not in the document]
Comment (stroblme) [reply app/app.py:14]: Hm, strange, could you try with a clean session, i.e. in a private tab?

[the remark this answers is not in the document]
Comment (stroblme) [reply app/app.py:14]: interesting.. in my case it's working fine.. if it's not working, then let's move the sidebar back into app.py

**Review by majafranz — APPROVED** [review]
Moved sidebar pages back to app

**Pull request merged**